tests/test.py

```diff
@@ -220,3 +221,70 @@ def test_edit_incident(post_from_slack_api, mock_slack):
     mock_slack.send_or_update_message_block.assert_called_with(
         "incident-channel-id", blocks=ANY, fallback_text=ANY, ts="123"
     )
+
+
+@pytest.mark.django_db(transaction=True)
+def test_edit_private_incident(post_from_slack_api, mock_slack):
+    mock_slack.send_or_update_message_block.return_value = {"ts": "123"}
+    mock_slack.get_user_profile.return_value = {"ts": "123", "name": "Opsy McOpsface"}
+
+    user = ExternalUser.objects.get_or_create(
+        app_id="slack", external_id="U123", display_name="Opsy McOpsface"
+    )[0]
+
+    incident = Incident.objects.create_incident(
+        report="Something happened",
+        reporter=user,
+        report_time=datetime.now(),
+        report_only=False,
+        private=True,
+        summary="Testing editing incidents - before",
+        impact="Lots",
+        lead=user,
+        severity="1",
+    )
+
+    newsummary = "Testing editing incidents - after"
+    data = {
```

Comment:
May be better to explicitly set private=False in this payload as that's being tested implicitly.